core/browser_events.ts

```diff
@@ -47,7 +51,7 @@ const PAGE_MODE_MULTIPLIER = 125;
 export function conditionalBind(
   node: EventTarget,
   name: string,
-  thisObject: Object | null,
+  thisObject: object | null,
```

Comment:
If I understand correctly this is considerably loosening the type of `thisObject`, which previously had to inherit from `Object.prototype` but can now be _any_ object.  Looks like that might be OK in this case, but something to be aware of.

Reply:
Yes it is. I agree it is ok.